### PR TITLE
[FIX] project,sale_project: fix the visibility of billable field and subtask assign button

### PR DESCRIPTION
**Pull request by gusi-odoo**
In the projectand sale_project module, Project Sharing  sub tasks view's notebook, hide the Billable field which is an
read-only field and in project kanban view assign button is  not visible in the subtask

task-3374861

### CODE REVIEW

File: addons/sale_project/views/project_sharing_views.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
                 <field name="sale_line_id" string="Sales Order Item" attrs="{'invisible': [('partner_id', '=', False)]}" options='{"no_open": True}' readonly="1" context="{'create': False, 'edit': False, 'delete': False}"/>
             </xpath>
             <xpath expr="//field[@name='child_ids']/tree/field[@name='partner_id']" position="after">
-                <field name="allow_billable"/>
+                <field name="allow_billable" invisible="1"/>
```
Comment (kpt-odoo) [inline addons/sale_project/views/project_sharing_views.xml:28]: reminder: remove the whole xpath in master it's of no use ​

**Review by MissingNoShiny — COMMENTED** [review]
LGTM, thanks for your work :slightly_smiling_face:

**Review by auon-odoo — APPROVED** [review]
thank you :)

**Comment by xavierbol** [comment]
robodoo rebase-ff r+

**Pull request closed without merging**